appengine/pond_storage/get-ducks.py

```diff
@@ -15,14 +15,25 @@
 limitations under the License.
 """
 
-"""Get all ducks for the current user 
+"""Given a duck id get the duck. If no duck id exists get all ducks belonging to the user.
```

Comment:
Since we're using it for get a duck and get multiple ducks, I would like for the file to be called get.py and also to also match how the other files are named too.

Also to shorten:
Given a duck key get the duck, otherwise get all ducks belonging to the user.